community-themes.json

```diff
@@ -344,4 +344,12 @@
         "screenshot": "images/Screenshot1.png",
         "compatibleModes": ["light"]
     }

```

Comment:
Could you add a comma to preserve the validity of the JSON file please?

Reply:
Now it is ok. Sorry about that.